controllers/users.js

```diff
@@ -41,18 +37,14 @@ exports.update = function(req, res) {
 			if (err) {
 				return res.status(400);
 			} else {
-				var o = {
-					name: user.name,
-					meetings: user.meetings
-				};
-				res.json(o);
+				res.json(user);
 			}
 		});
 	});
 };
 
 exports.editProfile = function(req, res) {
-  User.findOne({_id: req.user})
+  User.findOne({user_id: req.user})
```

Comment:
This should actually be _id, not user_id, since the field user_id does not exist on the user schema.